font/src/ft/fc/pattern.rs

```diff
@@ -537,6 +540,10 @@ impl PatternRef {
         }
     }
 
+    pub fn hash(&self) -> PatternHash {
+        unsafe { PatternHash(FcPatternHash(self.as_ptr())) }
+    }
```

Comment:
A bit unfortunate that Rust's `Hash` function doesn't just return a u32 or something and we can use that ourselves.

But I suppose this should work just fine.